crates/project_model/src/project_json.rs

```diff
@@ -37,7 +37,7 @@ impl ProjectJson {
     pub fn new(base: &AbsPath, data: ProjectJsonData) -> ProjectJson {
```

Comment:
Can you add a doc comment here that explains where `base` should point to when calling it?

Reply:
Certainly.  I found this by seeing what the resultant `pwd` to a started command was, and realizing that the path here is to the workspace folder that contains rust-project.json, not the path to rust-project.json itself.

Reply:
Done.